eZ/Publish/Core/Comparison/Engine/Value/StringValueComparisonEngine.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Comparison\Engine\Value;
+
+use eZ\Publish\API\Repository\Values\Content\VersionDiff\DataDiff\DiffStatus;
+use eZ\Publish\API\Repository\Values\Content\VersionDiff\DataDiff\StringDiff;
+use eZ\Publish\SPI\Comparison\Value\StringComparisonValue;
+use SebastianBergmann\Diff\Differ;
+
+final class StringValueComparisonEngine
+{
+    /** @var \SebastianBergmann\Diff\Differ */
+    private $innerEngine;
+
+    public function __construct()
+    {
+        $this->innerEngine = new Differ();
+    }
+
+    /**
+     * @return \eZ\Publish\API\Repository\Values\Content\VersionDiff\DataDiff\StringDiff[]
+     */
+    public function compareValues(StringComparisonValue $stringA, StringComparisonValue $stringB): array
+    {
+        if ($stringA->value === $stringB->value) {
+            return [new StringDiff(
+                $stringA->value,
+                DiffStatus::UNCHANGED
+            )];
```

Comment:
IMHO in this form is more readable: 

```suggestion
            return [
	            new StringDiff(
	                $stringA->value,
	                DiffStatus::UNCHANGED
	            )
            ];
```